src/main/scala/org/deeplearning4j/scalphagozero/board/GoBoard.scala

```diff
@@ -67,14 +67,17 @@ class GoBoard(val row: Int, val col: Int) {
   }
 
   private def removeString(goString: GoString): Unit =
-    for (point <- goString.stones) {
-      // Removing a string can create liberties for other strings.
-      for (neighbor <- neighborMap((point._1, point._1)) if grid.get(neighbor.toCoords).isDefined) {
-        val neighborString = grid.get(neighbor.toCoords)
-        if (neighborString.get.equals(goString))
-          this.replaceString(neighborString.get.withLiberty(Point(point._1, point._2)))
+    goString.stones.foreach { point =>
+      neighborMap((point._1, point._1)).foreach { neighbor =>
```

Comment:
is it normal that we pass `(point._1, point._1)` to `neighborMap` here ?

Reply:
omg, no! good catch.

Reply:
Would like me to fix this or you'll fix it ?